Sources/protoc-gen-swift/Docs.docc/spm-plugin.md

```diff
@@ -88,13 +88,26 @@ is generating Swift types for the `Bar.proto` file with the `public` visibility.
 
 ### Defining the path to the protoc binary
 
+
 The plugin needs to be able to invoke the `protoc` binary to generate the Swift types. 
-There are two ways how this can be achieved. First, by default, the package manager looks into
+There are three ways how this can be achieved. First, by default, the package manager looks into
 the `$PATH` to find binaries named `protoc`. This works immediately if you use `swift build` to build
 your package and `protoc` is installed in the `$PATH` (`brew` is adding it to your `$PATH` automatically).
 However, this doesn't work if you want to compile from Xcode since Xcode is not passed the `$PATH`.
-To still make this work from Xcode you can point the plugin to the concrete location of the `protoc`
-compiler by changing the configuration file like this:
+You have to options to set the path of `protoc` that the plugin is going to use. Either you can set
+an environment variable `PROTOC_PATH` that gets picked up by the plugin. Here are two example how you
+can set the variable so that it gets picked up:
+
+```shell
+// swift build
+env PROTOC_PATH=/opt/homebrew/bin/protoc swift build
+
+// To start Xcode (Xcode MUST NOT be running before invoking this)
+env PROTOC_PATH=/opt/homebrew/bin/protoc xed .
+```
```

Comment:
Should we mention (or example) anything about using it with `xcodebuild`?  Does the env var (and the `PATH` method) _just work_?

Reply:
Good point! `xcodebuild` works with the default `$PATH` as well, but the env var also works. I added the latter part as an example.